src/hotspot/cpu/x86/assembler_x86.cpp

```diff
@@ -1393,24 +1440,56 @@ void Assembler::addb(Address dst, int imm8) {
   emit_int8(imm8);
 }
 
+void Assembler::eaddb(Register dst, Address src, int imm8, bool no_flags) {
+  InstructionMark im(this);
+  InstructionAttr attributes(AVX_128bit, /* vex_w */ false, /* legacy_mode */ false, /* no_mask_reg */ true, /* uses_vl */ false);
+  attributes.set_address_attributes(/* tuple_type */ EVEX_NOSCALE, /* input_size_in_bits */ EVEX_8bit);
+  evex_prefix_ndd(src, dst->encoding(), 0, VEX_SIMD_NONE, VEX_OPCODE_0F_3C, &attributes, no_flags);
+  emit_int8((unsigned char)0x80);
+  emit_operand(rax, src, 1);
+  emit_int8(imm8);
+}
+
 void Assembler::addb(Address dst, Register src) {
   InstructionMark im(this);
   prefix(dst, src);
   emit_int8(0x00);
   emit_operand(src, dst, 0);
 }
 
+void Assembler::eaddb(Register dst, Address src1, Register src2, bool no_flags) {
+  InstructionAttr attributes(AVX_128bit, /* vex_w */ false, /* legacy_mode */ false, /* no_mask_reg */ true, /* uses_vl */ false);
+  attributes.set_address_attributes(/* tuple_type */ EVEX_NOSCALE, /* input_size_in_bits */ EVEX_8bit);
+  evex_prefix_ndd(src1, dst->encoding(), src2->encoding(), VEX_SIMD_NONE, VEX_OPCODE_0F_3C, &attributes, no_flags);
+  emit_int8(0x00);
+  emit_operand(src2, src1, 0);
+}
+
 void Assembler::addb(Register dst, int imm8) {
   (void) prefix_and_encode(dst->encoding(), true);
   emit_arith_b(0x80, 0xC0, dst, imm8);
 }
 
+void Assembler::eaddb(Register dst, Register src, int imm8, bool no_flags) {
+  InstructionAttr attributes(AVX_128bit, /* vex_w */ false, /* legacy_mode */ false, /* no_mask_reg */ true, /* uses_vl */ false);
+  // (void) evex_prefix_and_encode_ndd(src->encoding(), dst->encoding(), 0, VEX_SIMD_NONE, VEX_OPCODE_0F_3C, &attributes);
```

Comment:
Looks like the commented line is left over.

Reply:
Thanks, done.